src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.search__form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search__input {
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px
+    rgba(61, 78, 97, 0.1);
```

Comment:
add `font-family: inherit` for inputs
Inputs can't inherit font-family from the html by default, we should add this rule for inputs